garak/cli.py

```diff
@@ -3,7 +3,7 @@
 
 """Flow for invoking garak from the command line"""
 
-command_options = "list_detectors list_probes list_generators list_buffs list_config plugin_info interactive report version".split()
+command_options = "list_detectors list_probes list_generators list_buffs list_config plugin_info interactive report version skip_unknown".split()
```

Comment:
Usually, `command_options` lists things that determine the main part of what garak will do in this execution. `skip_unknown` seems more like a config option, which shouldn't be stored here. Later, things in `command_options` are handled separately from config, which also seems undesirable.

Reply:
Moving this to be a `system_params` value. We discussed separately that this option only really makes sense in the context of validation of overall configuration in the `cli` class, however after some review this may be analogous with `parallel_*` and `verbose` as a behavior of the overall system config.